.github/workflows/build-test.yaml

```diff
@@ -34,6 +34,7 @@ jobs:
       - name: Run formatting checks
         run: |
           cargo fmt --all -- --check
+          dprint fmt
```

Comment:
I think we should run `$   dprint check`

Reply:
indeed better.